packages/client/src/runtime/core/engines/client/ClientEngine.ts

```diff
@@ -308,8 +288,8 @@ function getErrorMessageWithLink(engine: ClientEngine, title: string) {
     binaryTarget: undefined,
     title,
     version: engine.config.clientVersion!,
-    engineVersion: engine.versionInfo?.commit,
+    engineVersion: 'unknown', // WASM engines do not export their version info
```

Comment:
ℹ️ Not sure if this something we want to eventually change but for WASM the underlying methods used by the `LibraryEngine` all just report `unknown`. 🤷